### PR TITLE
Add typescript support

### PR DESCRIPTION
**Pull request by clark-john**
D ko pa po binago yung version actually hahaha, kse baka ikaw na mag-aadjust nun

### CODE REVIEW

File: fileLogger.ts [hunk 1]
```diff
@@ -1,25 +1,35 @@
-import winston, { createLogger } from "winston";
+import winston, { createLogger, Logger } from "winston";
```
Comment (albertdiones) [inline fileLogger.ts:1]: hindi ba to magcoconflict sa Logger sa logger.ts?

File: tsconfig.json
Comment (albertdiones) [inline tsconfig.json:1]: nice

**Review by albertdiones — APPROVED** [review]
tested locally by cloning the fork

File: fileLogger.ts (hunk 1 above)
Comment (mjohnclark) [reply fileLogger.ts:1]: d naman po since nasa kabilang file naman yung Logger class mo po mismo

File: package.json [hunk 2]
```diff
@@ -2,24 +2,32 @@
...
   },
   "keywords": [
     "logger"
   ],
-  "author": "Albert Diones",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/albertdiones/logger/issues"
+  "files": [
+    "./dist/*.js"
```
Comment (albertdiones) [inline package.json:20]: bakit js to instead of ts? also non existent directory ./dist/

Comment (mjohnclark) [reply package.json:20]: kung i-bbuild mo sya into js then magkakaroon sya ng dist folder and publish it

**Pull request merged**